Framework/Core/include/Framework/SendingPolicy.h

```diff
@@ -24,7 +25,7 @@ namespace o2::framework
 class FairMQDeviceProxy;
 
 struct SendingPolicy {
-  using SendingCallback = std::function<void(FairMQDeviceProxy&, fair::mq::Parts&, ChannelIndex channelIndex)>;
+  using SendingCallback = std::function<void(FairMQDeviceProxy&, fair::mq::Parts&, ChannelIndex channelIndex, ServiceRegistry& registry)>;
```

Comment:
This change in the API makes sense in any case. Can you take care of it in a separate API, so that we get it out of the way immediately?